spec/spec_helper.rb

```diff
@@ -1,6 +1,15 @@
-ENV["LD_LIBRARY_PATH"] ||= File.expand_path("ext", __dir__)
```

Comment:
I think this line was here for the Mac, but on newer Mac versions, it doesn't do anything anyway because of the SIP stuff.

Even so, I expected this to work on Linux, and I have no idea why you need to pass LD_LIBRARY_PATH manually.  I guess that's something we can solve another day.